src/snmalloc/backend/backend.h

```diff
@@ -57,7 +57,8 @@ namespace snmalloc
           GlobalMetaRange::ConcurrencySafe,
           "Global meta data range needs to be concurrency safe.");
         GlobalMetaRange global_state;
-        p = global_state.alloc_range(bits::next_pow2(size));
+        auto rsize = bits::max(bits::next_pow2(size), MIN_CHUNK_SIZE);
+        p = global_state.alloc_range(rsize);
```

Comment:
Don't we use `alloc_meta_data` for very tiny `Metaslab` objects?  Do they never hit this `else` branch?

Reply:
Those cases will normally have a LocalState.  The only snmalloc use for this is allocating the allocator.

Reply:
Maybe we can have some kind of assert that we don't hit this path for something, say, smaller than `MIN_CHUNK_SIZE / 4` or something reasonable in that neighborhood?

Reply:
Makes sense.